recipe/meta.yaml

```diff
@@ -63,3 +63,6 @@ extra:
   recipe-maintainers:
     - isuruf
     - jakirkham
+    - kkraus14
+    - mike-wendt
+    - raydouglass
```

Comment:
@kkraus14 @mike-wendt @raydouglass, would you be ok being added as maintainers on the `nvcc` (shim compiler package) feedstock? Please let me know 🙂

Reply:
@jakirkham would be my pleasure! Suggest adding @dillon-cullinan as well.

Reply:
@jakirkham same, happy to help and support @dillon-cullinan being added as well

Reply:
Thanks all! 😄 Have added Dillon as well. 👍